crates/sui-storage/src/object_store/util.rs

```diff
@@ -198,6 +212,45 @@ pub async fn find_all_dirs_with_epoch_prefix(
     Ok(dirs)
 }
 
+pub async fn list_all_epochs(object_store: Arc<DynObjectStore>) -> Result<Vec<u64>> {
+    let remote_epoch_dirs = find_all_dirs_with_epoch_prefix(&object_store, None).await?;
+    let mut out = vec![];
+    for (epoch, path) in remote_epoch_dirs.iter() {
+        let success_marker = path.child("_SUCCESS");
+        let get_result = object_store.get(&success_marker).await;
+        match get_result {
+            Err(_) => {
+                error!("No success marker found for epoch: {epoch}");
+            }
```

Comment:
This is really only an error when the epoch in question is not the last epoch in the sequence right? 

In such a case, it might be good to bump a counter so we can know that this has happened (as this should never happen unless we messed something up or deleted data). 

In the case where it's the last epoch of the sequence, this could just be because it's in the process of being uploaded right? So erroring would pollute the logs every 5 minutes until then